tesseract-core/src/lib.rs

```diff
@@ -654,15 +654,24 @@ impl Schema {
 
         let sort = if let Some(ref s) = query.sort {
             // sort column needs to be named by alias
-            let sort_column = query.measures.iter()
-                .position(|m| m == &s.measure)
-                .ok_or_else(|| format_err!("sort {:?} not found in measures", &s.measure))?;
+            // Checking if we are going to sort by RCA or not
+            let measure_name = &s.measure.to_string();
+            if measure_name.contains("RCA") {
```

Comment:
The only thing I wonder is, should we do a more specific check here that the sort is some form of "<Column> RCA"? because now if you did `sort=Blah RCA` it would catch it even if it's not an actual column. It might just help avoid future issue in case someone has a column named like `RCAP` and they want to sort on that

Reply:
Will take a look at it